### DISCUSSION TITLE
en/release/v2

### DISCUSSION THREAD

**Discussion by giscus** [gogf/gf-site#92] [General]
# en/release/v2

This version update includes several important improvements, including the minimum Golang version requirement adjustment from 1.18 to 1.20, the deprecation of the gring component, the removal of the jaeger component, improvements in gjson parameter forms, and increased flexibility for ghttp and gudp. New features include database connection support for unix socket, time field type support, the addition of the Exist method in gdb, enhanced gconv conversion performance, improved gvalid validation rules and gtest assertion methods, enhanced gcron task management, and significantly improved functionality and extensibility of GoFrame development tools.

https://goframe.org/en/release/v2.8

**Comment by hailaz** [comment]
hello

**Comment by ninjashixuan** [comment]
contrib/drivers/pgsql  scan 可以增加 pg 数组到 golang 数组吗，现在scan 要额外 type 然后现实UnmarshalValue 不是很方便，希望内置支持。

**Reply by ninjashixuan** [reply]
也希望 gf gen dao 生成 entity 时也 pg 数组 直接是 []type。

**Reply by gqcn** [reply]
非常欢迎参与共建哈，我们社区对开源贡献都是非常开放的。❤️

**Reply by ninjashixuan** [reply]
试了一下，[]int 类型 gf gen dao 和 scan 都已经实现了，[]string 还没有，我翻翻代码看看能不能提个 pr。
